README.md

```diff
@@ -120,6 +129,44 @@ Some parameters are common to every event:
 }
 ```
 
+# FAQ
+
+## Do you have a dashboard/status page?
+
+Not currently.
+
+## Can you please share the delivery delay for the webhook?
+
+Indexing can take 1-2 minutes in the worst cases and less than 15 seconds in good cases.
+
+## Will the webhooks do retries?
+
+Currently no, and please count on that maybe due to some network issues you can lose a webhook. We will work on resilience patters like retrying or removing an integration if service cannot deliver webhooks for some time.
+
+## Do you plan to have a way to trigger a backfill in case our systems go down?
+
+In case our systems go down, messages should be stored in our queue and when the systems are up resending should be restored (unless queue is overflowed because services have been done for a while and some old messages are discarded).
+
+## Is it available on all chains already?
+
+Yes, and we can configure the chains you want to get events from.
```

Comment:
Maybe clarify that all the official supported chains and link to them https://docs.safe.global/safe-core-api/supported-networks

Reply:
Done, you can review now